tests/rt.conf

```diff
@@ -120,7 +122,7 @@ RUN     | control_thompson_progcld_thompson_debug
 
 RUN     | regional_debug                                                                                                          |                                         | fv3 |
 
-COMPILE | -DAPP=ATM -DDEBUG=ON -DCCPP_SUITES=FV3_RAP,FV3_RAP_cires_ugwp,FV3_RAP_unified_ugwp,FV3_RAP_flake -D32BIT=ON             |                                         | fv3 |
+COMPILE | -DAPP=ATM -DDEBUG=ON -DCCPP_SUITES=FV3_RAP,FV3_RAP_cires_ugwp,FV3_RAP_unified_ugwp,FV3_RAP_flake -D32BIT=ON             | + wcoss_dell_p3 wcoss_cray              | fv3 |
```

Comment:
Why do we add dell/cray in the platform list? Does it mean this compile job only works on dell/cray?

Reply:
With WCOSS dell/cray going away soon, maybe we should remove this?